### PR TITLE
Create de.lua

### PR DESCRIPTION
**Pull request by Seraph1982**
de.lua for adminmenu

### CODE REVIEW

**Review by jay-fivem — APPROVED** [review]
Approved

File: locales/de.lua [hunk 1]
```diff
@@ -0,0 +1,222 @@
+local Translations = {
+    error = {
+        ["blips_deactivated"] = "Blips deaktiviert",
+        ["names_deactivated"] = "Names deactiviert",
```
Comment (Tomboru) [inline locales/de.lua:4]:
```suggestion
        ["names_deactivated"] = "Namen deaktiviert",
```

File: locales/de.lua [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+        ["changed_perm_failed"] = "suche eine Gruppe aus!",
```
Comment (Tomboru) [inline locales/de.lua:5]:
```suggestion
        ["changed_perm_failed"] = "Wähle eine Gruppe aus!",
```

File: locales/de.lua [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+        ["no_weapon"] = "Du hast keine Waffe in deiner Hand..",
+        ["no_free_seats"] = "Dieses Fahrzeug hat keinen freien Sitz!",
+        ["failed_vehicle_owner"] = "Das Fahrzeug ist schon deins..",
+        ["not_online"] = "Dieser Spieler ist nicht online",
+        ["no_receive_report"] = "Du bekommst keine Reports",
+        ["failed_set_speed"] = "Du hast kein speed festgelegt.. (`fast` for super-run, `normal` for normal)",
+        ["failed_set_model"] = "Du hast kein model festgelegt..",
+    },
+    success = {
+        ["blips_activated"] = "Blips activiert",
```
Comment (Tomboru) [inline locales/de.lua:19]:
```suggestion
        ["blips_activated"] = "Blips aktiviert",
```

File: locales/de.lua [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+        ["names_activated"] = "Names activiert",
```
Comment (Tomboru) [inline locales/de.lua:20]:
```suggestion
        ["names_activated"] = "Namen aktiviert",
```

File: locales/de.lua [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+    },
+    info = {
+        ["ped_coords"] = "Ped Koordinaten:",
+        ["vehicle_dev_data"] = "Fahrzeug Developer Data:",
+        ["ent_id"] = "Entity ID:",
+        ["net_id"] = "Net ID:",
+        ["model"] = "Model",
+        ["hash"] = "Hash",
+        ["eng_health"] = "Motor Health:",
+        ["body_health"] = "Karrosierie Health:",
```
Comment (Tomboru) [inline locales/de.lua:36]:
```suggestion
        ["body_health"] = "Body Health:",
```

File: locales/de.lua [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+        ["goto"] = "Gehe zu", 
+        ["remove"] = "Entfernen",
+        ["confirm"] = "Bestätigen",
+        ["reason_title"] = "Grund",
+        ["length"] = "Länge",
+        ["options"] = "Optionen",
+        ["position"] = "Position",
+        ["your_position"] = "zu deiner position",
```
Comment (Tomboru) [inline locales/de.lua:44]:
```suggestion
        ["your_position"] = "Zu dir",
```

File: locales/de.lua [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+        ["open"] = "Öffne",
+        ["inventories"] = "Inventar",
+        ["reason"] = "Du musst einen Grund angeben", 
+        ["give"] = "gib",
```
Comment (Tomboru) [inline locales/de.lua:48]:
```suggestion
        ["give"] = "Geben",
```

File: locales/de.lua [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+        ["rank_level"] = "Dein Rechte Level ist jetzt ",
+        ["admin_report"] = "Admin Report - ",
+        ["staffchat"] = "STAFFCHAT - ",
+        ["warning_chat_message"] = "^8WARNING ^7 Du wurdest verwarnt durch",
+        ["warning_staff_message"] = "^8WARNING ^7 Du hast jemanden verwarnt ",
+        ["no_reason_specified"] = "Keinen Grund spezifizert",
+        ["server_restart"] = "Server restart, prüfe unseren Discord für mehr Informationen: ",
+    },
+    menu = {
+        ["admin_menu"] = "Admin Menu",
```
Comment (Tomboru) [inline locales/de.lua:68]:
```suggestion
        ["admin_menu"] = "Admin Menü",
```

File: locales/de.lua [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+        ["admin_options"] = "Admin Optionen",
+        ["online_players"] = "Online Spieler",
```
Comment (Tomboru) [inline locales/de.lua:70]:
```suggestion
        ["online_players"] = "Spieler online",
```

File: locales/de.lua [hunk 10]
```diff
@@ -0,0 +1,222 @@
...
+        ["manage_server"] = "Manage Server",
+        ["weather_conditions"] = "Vorhandene Wetter Optionen",
```
Comment (Tomboru) [inline locales/de.lua:72]:
```suggestion
        ["weather_conditions"] = "Wetter Optionen",
```

File: locales/de.lua [hunk 11]
```diff
@@ -0,0 +1,222 @@
...
+        ["dealer_list"] = "Dealer Liste",
+        ["ban"] = "Ban",
+        ["kick"] = "kick",
```
Comment (Tomboru) [inline locales/de.lua:75]:
```suggestion
        ["kick"] = "Kick",
```

File: locales/de.lua [hunk 12]
```diff
@@ -0,0 +1,222 @@
...
+        ["permissions"] = "Rechte",
+        ["developer_options"] = "Developer Optionen",
+        ["vehicle_options"] = "Fahrzeug Optionen",
+        ["vehicle_categories"] = "Fahrzeug Kategorien",
+        ["vehicle_models"] = "Fahrzeug Modelle",
+        ["player_management"] = "Spieler Management",
+        ["server_management"] = "Server Management",
+        ["vehicles"] = "Fahrzeuge",
+        ["noclip"] = "NoClip",
+        ["revive"] = "Revive",
```
Comment (Tomboru) [inline locales/de.lua:85]:
```suggestion
        ["revive"] = "Wiederbeleben",
```

File: locales/de.lua [hunk 13]
```diff
@@ -0,0 +1,222 @@
...
+        ["server_time"] = "Server Zeit",
+        ["time"] = "Zeit",
+        ["copy_vector3"] = "Kopiere vector3", 
+        ["copy_vector4"] = "Kopiere vector4",
+        ["display_coords"] = "Zeige Koordinaten",
+        ["copy_heading"] = "Kopiere Heading",
+        ["vehicle_dev_mode"] = "Fahrzeug Dev Mode",
+        ["delete_laser"] = "Lösch Laser", 
+        ["spawn_vehicle"] = "Spawne Fahrzeug",
+        ["fix_vehicle"] = "Fixe Fahrzeug",
```
Comment (Tomboru) [inline locales/de.lua:100]:
```suggestion
        ["fix_vehicle"] = "Fahrzeug reparieren",
```

File: locales/de.lua [hunk 14]
```diff
@@ -0,0 +1,222 @@
...
+        ["buy"] = "Kaufe",
+        ["remove_vehicle"] = "Entferne Fahrzeug",
+        ["edit_dealer"] = "Editiere Dealer ", 
+        ["dealer_name"] = "Dealer Name",
+        ["category_name"] = "Kategorie Name",
+        ["kill"] = "Kill",
+        ["freeze"] = "Friere",
```
Comment (Tomboru) [inline locales/de.lua:107]:
```suggestion
        ["freeze"] = "Einfrieren",
```

File: locales/de.lua [hunk 15]
```diff
@@ -0,0 +1,222 @@
...
+        ["spectate"] = "beobachten",
```
Comment (Tomboru) [inline locales/de.lua:108]:
```suggestion
        ["spectate"] = "Beobachten",
```

File: locales/de.lua [hunk 16]
```diff
@@ -0,0 +1,222 @@
...
+        ["bring"] = "Bringe",
```
Comment (Tomboru) [inline locales/de.lua:109]:
```suggestion
        ["bring"] = "Herbringen",
```

File: locales/de.lua [hunk 17]
```diff
@@ -0,0 +1,222 @@
...
+        ["give_clothing_menu"] = "Gib Kleidungs Menu",
+    },
+    desc = {
+        ["admin_options_desc"] = "Verschiedene. Admin Optionen",
+        ["player_management_desc"] = "Siehe Spieler Optionen",
+        ["server_management_desc"] = "Verschiedene Server Optionen",
+        ["vehicles_desc"] = "Fahrzeug Optionen",
+        ["dealer_desc"] = "Liste Existierender Dealer",
+        ["noclip_desc"] = "An/Aus NoClip",
+        ["revive_desc"] = "Revive dich selbst",
```
Comment (Tomboru) [inline locales/de.lua:121]:
```suggestion
        ["revive_desc"] = "Wiederbelebe dich selbst",
```

**Review by Tomboru — CHANGES_REQUESTED** [review]

**Comment by FjamZoo** [comment]
Please update your PR with the suggested changes from Tomboru and this can be pulled.

**Pull request merged**